src/sentry/models/distribution.py

```diff
@@ -0,0 +1,57 @@
+"""
+sentry.models.distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from django.db import models, IntegrityError, transaction
+from django.utils import timezone
+
+from sentry.db.models import Model, FlexibleForeignKey, sane_repr
+
+
+class Distribution(Model):
+    __core__ = False
+
+    release = FlexibleForeignKey('sentry.Release')
+    name = models.CharField(max_length=64)
+    date_added = models.DateTimeField()
```

Comment:
Should just do `default=timezone.now` here, then you can avoid it in the `get_or_create` below.